application/src/main/java/org/togetherjava/tjbot/features/tags/TagCommand.java

```diff
@@ -85,8 +84,6 @@ public void onSlashCommand(SlashCommandInteractionEvent event) {
 
         String tagContent = tagSystem.getTag(id).orElseThrow();
         MessageEmbed contentEmbed = new EmbedBuilder().setDescription(tagContent)
-            .setFooter(event.getUser().getName() + " • used " + event.getCommandString())
-            .setTimestamp(Instant.now())
```

Comment:
please keep that one. it was put there on purpose to educate people new to discord (50% of our user base) how to use it themselves. they are not discord power users

Reply:
Done